Automattic/jetpack
# Slideshow: Move and resize pagination dots

Closes #27708

#### Changes proposed in this Pull Request:

* Reduce `bullet` size from `16px` to `8px`.
* Move `bullets` from `center` to `flex-start`.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Create a JN site
* Connect Jetpack
* Create a new post with a Slideshow
* Insert 5 or fewer images (to make sure will appear the dots and not the number).
* Visit the post with the slideshow
* Check if the dots are smaller and if it was moved to the beginning.

#### Demo 💻 

![image](https://user-images.githubusercontent.com/1663717/208465148-e5c21a5f-62db-4653-a49b-53572d2dfd09.png)
